recipes/mikeio/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "mikeio" %}
+{% set version = "0.5.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5e71b648c73bb82fdf73641ac5272fef7eab0ca637b53631d28e22a66e04617d
+  
+build:
+  
+  #noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36] [not win]"
```

Comment:
This recipe should be noarch python, you can restrict python below with `python >=3.6`